adb_graphics/specs.py

```diff
@@ -197,7 +197,24 @@ def lcl_colors(self) -> np.ndarray:
         ctable = ctables.colortables.get_colortable(self.vspec.get('cmap')) \
                     (range(60, 180, 6)) # rainbow
         return ctable
+    @property
+    @lru_cache()
+    def mdn_colors(self) -> np.ndarray:
+
+        ''' Default color map for Max Downdraft '''
 
+        grays = cm.get_cmap('Greys', 2)([0])
+        jet18 = cm.get_cmap('jet', 18)(range(18, 1, -1), alpha=0.6)
```

Comment:
The colormap should be set in the default_specs.yml file and read in here. See lifted_index_colors below for an example.

When choosing names for variables, please choose them so that they are generic and do not hint at the contents of one instance of the variable. The variable `ctable` is used in many of the other configurable color tables described here.